### PR TITLE
Loss fn

### PR DESCRIPTION
**Pull request by seemuch**

### CODE REVIEW

File: official/resnet/keras/keras_imagenet_main.py [hunk 1]
```diff
@@ -250,6 +250,18 @@ def softmax_crossentropy_with_logits(y_true, y_pred):
       logits=y_pred,
       labels=tf.reshape(tf.cast(y_true, tf.int64), [-1,]))
 
+def cross_entropy_plus_l2_loss(y_true, y_pred):
+  weight_decay = 1e-4
+  entropy_loss = _softmax_crossentropy_with_logits(y_true, y_pred)
+  l2_loss = weight_decay * tf.add_n(
+      # loss is computed using fp32 for numerical stability.
+      [tf.nn.l2_loss(tf.cast(v, tf.float32)) for v in tf.trainable_variables()
```
Comment (anj-s) [inline official/resnet/keras/keras_imagenet_main.py]: Is this populated for Keras? I think we may need to add regularization losses to the layers explicitly. What is the difference in loss that you see with this change?
